mediaflow_proxy/extractors/dlhd.py

```
@@ -1,9 +1,13 @@
 import re
-from typing import Dict, Any, Optional
-from urllib.parse import urlparse, quote
+import base64
+import logging
+from typing import Any, Dict, Optional
+from urllib.parse import urlparse, quote, urlunparse
```

Ruff (0.12.2)

`urllib.parse.urlunparse` imported but unused

Remove unused import: `urllib.parse.urlunparse`

(F401)